as-clients/rfp.md

```diff
@@ -0,0 +1,93 @@
+Atlantean Banking: Request for Proposals
+========================================
+
+## Document History
+
+| Version | When            | Who                    | What                                      |
+| ---     | ---             | ---                    | ---                                       |
+| v1.0    | 2023-01-24      | Client team            | Initial drafting                          |
+
+## Table of Contents
+
+(1.0) Product Overview  
+(2.0) Project Objectives  
+(3.0) Current System(s)  
+(4.0) Intended Users of the System  
+(5.0) Known interactions with systems within or outside the client organization  
+(6.0) Known constraints to development  
+(7.0) Project Schedule  
+(8.0) Project team  
+(9.0) Glossary of terms  
+
+## (1.0) Product Overview
+
+We the people of “Atlantis” have resurfaced from the depths of the ocean. We have our own currency and bank but need an online banking system to reconnect with the world. The issues we have are the following:
+
+- Unable to e-transfer other banking accounts
+- Unable to view avaible funds
+- No online payment methods
+- The city of Atlantis is growing and visiting the bank is not scalable. Customers having to go to the bank just to view the transaction history is causing problems. 
+
+We predict that making an online banking system will solve these issues and increase customer satisfaction for all their banking needs. 
+
+## (2.0) Project Objectives
+
+The (name of bank) online banking system needs to fulfill the following needs
```

Comment:
We need to replace this. We can even avoid mentioning a bank name at all, see suggestion
```suggestion
The online banking system needs to fulfil the following needs
```